utils/rpms/packaging/rpm_chrootbuild

```diff
@@ -4,16 +4,21 @@ set -uex
 
 # shellcheck disable=SC2153
 IFS=\| read -r -a distro_base_local_repos <<< "$DISTRO_BASE_LOCAL_REPOS"
+repo_adds=()
+repo_dels=()
 
-if [ -w /etc/mock/"$CHROOT_NAME".cfg ]; then
-    # Remove the distro repos
-    if [[ $CHROOT_NAME =~ opensuse-leap-* ]]; then
-        ed /etc/mock/"$CHROOT_NAME".cfg << EOF
-/^# repos$/+2;/^"""$/-1d
-wq
-EOF
-    fi
-    echo -e "config_opts['yum.conf'] += \"\"\"\n" >> /etc/mock/"$CHROOT_NAME".cfg
+if [ -n "$REPOSITORY_URL" ] && [ -n "$DISTRO_REPOS" ]; then
+    repo_dels+=("--disablerepo=\*")
+fi
+
+original_cfg_file="/etc/mock/${CHROOT_NAME}.cfg"
+cfg_file="/etc/mock/${CHROOT_NAME}_daos.cfg"
```

Comment:
Should put this somewhere else, where we can write, and use `mock`'s `--configdir=CONFIGDIR` to point at it.

Reply:
I did not see the --configdir option.  I will need to try that.

Reply:
Fixed in current push.